tests/mockoon_tpay.json

```diff
@@ -841,4 +841,4 @@
   ],
   "data": [],
   "callbacks": []
-}
+}
```

Comment:
I’ve experienced this as well. I think we should keep this change since Mockoon doesn’t seem to handle blank lines in the configuration well 😄